src/super_gradients/training/models/__init__.py

```diff
@@ -135,51 +135,28 @@
 from super_gradients.training.utils import make_divisible as _make_divisible_current_version, HpmStruct as CurrVersionHpmStruct
 
 
-def make_deprecated(func, reason):
-    def inner(*args, **kwargs):
-        with warnings.catch_warnings():
-            warnings.simplefilter("once", DeprecationWarning)
-            warnings.warn(reason, category=DeprecationWarning, stacklevel=2)
-        warnings.warn(reason, DeprecationWarning)
-        return func(*args, **kwargs)
+@deprecated(deprecated_in_v="3.1.0", remove_in_v="3.4.0", target=_make_divisible_current_version)
```

Comment:
Minor complain, but why `deprecated_in_v` and not `deprecated_in_version` or `deprecated_since` ?
Similarly - `remove_in_v` -> `removed_in_version`/`removed_from` maybe?

Reply:
Updated in [65fc918](https://github.com/Deci-AI/super-gradients/pull/1416/commits/65fc91897a13e5f55abd9408f312c5ab4aefc8e7)